frontend/app/templates/index.hbs

```diff
@@ -0,0 +1,8 @@
+{{#link-to 'tasks.new' class="btn btn-primary btn-lg"}}
+  Create
```

Comment:
Good point. I don't have much experience with it on ember. Will have a look and see if it's easy to implement, although I might do it in a different PR